frontend/src/components/redpanda-ui/components/breadcrumb.tsx

```diff
@@ -0,0 +1,116 @@
+import { ChevronRight, MoreHorizontal } from 'lucide-react';
+import { Slot as SlotPrimitive } from 'radix-ui';
+import React from 'react';
+
+import { cn } from '../lib/utils';
+
+function Breadcrumb({ testId, ...props }: React.ComponentProps<'nav'> & { testId?: string }) {
+  return <nav aria-label="breadcrumb" data-slot="breadcrumb" data-testid={testId} {...props} />;
+}
+
+function BreadcrumbList({ className, testId, ...props }: React.ComponentProps<'ol'> & { testId?: string }) {
```

Comment:
question: maybe these would belong to a UI-registry in the future?

Reply:
actually this came from registry originally :)